Makefile

```diff
@@ -13,7 +13,7 @@ DEIS_REGISTRY ?= ${DEV_REGISTRY}
 RC := manifests/deis-${SHORT_NAME}-rc.yaml
 SVC := manifests/deis-${SHORT_NAME}-service.yaml
 SEC := manifests/deis-${SHORT_NAME}-secretAdmin.yaml
-IMAGE := ${DEIS_REGISTRY}/${SHORT_NAME}:${VERSION}
+IMAGE := ${DEIS_REGISTRY}/${IMAGE_PREFIX}/${SHORT_NAME}:${VERSION}
```

Comment:
agreed, and I've been following that discussion. are we standardizing on something? otherwise, I'd like to leave as-is until we do